superset/reports/notifications/email.py

```diff
@@ -38,6 +39,31 @@
 TABLE_TAGS = ["table", "th", "tr", "td", "thead", "tbody", "tfoot"]
 TABLE_ATTRIBUTES = ["colspan", "rowspan", "halign", "border", "class"]
 
+ALLOWED_TAGS = [
+    "a",
+    "abbr",
+    "acronym",
+    "b",
+    "blockquote",
+    "br",
+    "code",
+    "div",
+    "em",
+    "i",
+    "li",
+    "ol",
+    "p",
+    "strong",
+    "ul",
+] + TABLE_TAGS
```

Comment:
This is basically `bleach.ALLOWED_TAGS + ["br", "p", "div"]`. Enumerate all of them so you don't have to jump to the source code of `bleach` to see the full list.